src/test/resources/expected-classes/kt/AddLabelsToLabelableInput.kt.txt

```diff
@@ -0,0 +1,28 @@
+package com.github.graphql
+
+import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLRequestSerializer
+import java.util.StringJoiner
+
+@javax.annotation.Generated(
+    value = ["com.kobylynskyi.graphql.codegen.GraphQLCodegen"],
+    date = "2020-12-31T23:59:59-0500"
+)
+data class AddLabelsToLabelableInput(
+    val clientMutationId: String?,
+    val labelIds: List<String?>,
+    val labelableId: String?
+) {
+
+    // In the future, it maybe change.
+    override fun toString(): String {
+        val joiner = StringJoiner(", ", "{ ", " }")
+        if (clientMutationId != null) {
+            joiner.add("clientMutationId: " + GraphQLRequestSerializer.getEntry(clientMutationId))
+        }
+        joiner.add("labelIds: " + GraphQLRequestSerializer.getEntry(labelIds))
+        if (labelableId != null) {
+            joiner.add("labelableId: " + GraphQLRequestSerializer.getEntry(labelableId))
+        }
+        return joiner.toString()
+    }
```

Comment:
Why do we generate a `toString()` method here? Kotlin's `data class` already has one.

Reply:
This is used to process graphql query and must be rewritten

Reply:
Ah I see, so the actual string has meaning, and is not "just a `toString`", right?
In that case I understand.
I guess we should rename the method instead of using `toString` for business logic.

Reply:
This may need to be modified greatly. It is suggested that it should not be changed at first